src/style.css

```diff
@@ -1 +1,50 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+main {
+  width: 96px;
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('./images/star.svg');
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
+}
```

Comment:
```suggestion
.stars--0 .stars__star:nth-child(-n + 0),
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4).
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url('./images/star-active.svg');
}
```

Reply:
![image](https://github.com/mate-academy/layout_stars/assets/148764306/8b64ed28-2de8-49c8-9e5c-cc118b68b312)

Reply:
I can't push it with this errors

Reply:
It seems like you didn't change the selectors in appropriate way like I described in previous review.